docs/core/install/includes/linux-install-dependencies.md

```diff
@@ -0,0 +1,22 @@
+
+When you install with a package manager, these libraries are installed for you. But, if you manually install .NET Core or you publish a self-contained app, you'll need to make sure these libraries are installed:
+
+- lttng-ust
```

Comment:
You should include links to what these libraries are, so that someone reading this can quickly spot-check to ensure they're ok with these additional dependent libraries being installed too.

Reply:
Good suggestion. However, I'm not sure what to link to. This list was taken from the release notes of .NET Core.

Reply:
I'd expect Linux users interested in this to know how to use their package manager to find more info:

```
$ apt info liblttng-ust0
Package: liblttng-ust0
Version: 2.11.0-1
Priority: extra
Section: libs
Source: ust
Origin: Ubuntu
Maintainer: Ubuntu Developers <ubuntu-devel-discuss@lists.ubuntu.com>
Original-Maintainer: Jon Bernard <jbernard@debian.org>
Bugs: https://bugs.launchpad.net/ubuntu/+filebug
Installed-Size: 802 kB
Depends: libc6 (>= 2.17), libnuma1 (>= 2.0.11), liburcu6 (>= 0.11.1), liblttng-ust-ctl4 (= 2.11.0-1)
Conflicts: liblttng-ust2
Replaces: liblttng-ust2
Homepage: https://lttng.org/
Download-Size: 160 kB
APT-Sources: http://archive.ubuntu.com/ubuntu focal/main amd64 Packages
Description: LTTng 2.0 Userspace Tracer (tracing libraries)
```

Not opposed to adding more info here, but not sure it's necessary.